packages/nvidia-container-toolkit/nvidia-oci-hooks-json

```diff
@@ -4,7 +4,7 @@
     "prestart": [
       {
         "path": "/usr/bin/nvidia-container-runtime-hook",
-        "args": ["nvidia-container-runtime-hook", "prestart"]
+        "args": ["nvidia-container-runtime-hook", "-config", "/etc/nvidia-container-runtime/config.toml", "prestart"]
```

Comment:
This flag isn't needed, since the default value in `nvidia-container-runtime` is `/etc/nvidia-container-runtime/config.toml`:

https://github.com/NVIDIA/nvidia-container-toolkit/blob/b3519fadc469229e920766c4be7fad0fb955fd35/cmd/nvidia-container-runtime-hook/hook_config.go#L16

Reply:
I'll remove it then, just wanted to be explicit to surface the change

Reply:
Done